developer/developer-faqs.md

```diff
@@ -3,6 +3,15 @@
 ## Installation
 
 ## General
+Q: What do I need to know to work with Reaction?
```

Comment:
I think @machikoyasuda is doing ### for Q's now so they stand out more

Reply:
@zenweasel  yes, I was wondering about that as well. The whole development FAQs would have to be changed, though.

Reply:
Also, the first, second and third-level headings are too similar. And worse, are useless when there's no question in it. I'm going to change that .

![screen shot 2017-11-29 at 13 05 08](https://user-images.githubusercontent.com/1733229/33374323-f8e92fa2-d505-11e7-9e78-6081167391f2.png)